monero-outreach-docs/translations/pt-br/PSA/files/learning/learning_pt-br.md

```diff
@@ -0,0 +1,18 @@
+# Busque por mais conhecimento
+
+No momento que escrevo esse texto, uma suposição que se faz é de que a maioria da comunidade Monero é composta por prioneiros da tecnologia, sendo assim mais experientes tecnicamente do que uma pessoa comum. O código do Monero vem sendo desenvolvido por algumas centenas de programadores, quantia que é muito maior do que a maioria dos projetos de cripto moedas. Especula-se que uma das razões pelas quais o preço da Monero não se aproxima ao do Bitcoin é porque a tecnologia por trás do Monero é difícil de entender e apreciar. Mas nós, os pioreiros, continuamos a avançar na esperança de que o Monero se tornará amplamente reconhecido e adotado. O Monero daqui a 100 anos ainda dependerá de pessoas, pessoas como você, trabalhando para proteger o ecossistema e avançar ainda mais o código. Esses são todos os motivos pelos quais recomendamos que as pessoas sempre busquem por conhecimento, ainda mais se elas não forem desenvolvedores. Você pode fortalecer sua "alfabetização sobre Monero" consumindo o maior número de informações que puder.
+
+Ao aprender mais sobre os recursos do Monero, seu código, o espírito código aberto e cypherpunk que o originou e maneiras de melhorar sua segurança e sua privacidade ao utilizá-lo, juntos, poderemos fortalecer ainda mais o ecosistema do Monero. Ao ter um bom entendimento de todos os aspéctos do Monero, você também será capaz de falar sobre ele e ajudará a desenvolver uma maior adoção. Você se sentirá mais confortável e mais confiante em armazenar, usar e transacionar XMR. Conhecimento é poder e ele pode mudar o mundo. Sempre busque por mais conhecimento... e faça sua própria pesquisa!
+
+### O futuro começa com você, aqui estão algumas leituras que poderão te ajudar a aprender mais sobre o Monero:
+
+**_[getmonero.org (Site oficial)](https://getmonero.org/)_**
```

Comment:
Just noticed the getmonero.org site is now available in Brazilian Portuguese (pt-br). Now is a good chance to add the Portuguese links to the translations, so I will suggest changing starting with these ones. And this section needs to add a (+) sign at the beginning of every link to be a list.

Links to change:
- https://www.getmonero.org/pt-br/
- https://www.getmonero.org/pt-br/get-started/faq/

And by adding the (+) it should look like this:
+ **_[getmonero.org (Site oficial)](https://www.getmonero.org/pt-br/)_**
+ **_[monero.how](https://www.monero.how/)_**
+ **_[reddit.com/r/monero](https://www.reddit.com/r/Monero/)_**
+ **_[Guia do Monero (postagem)](https://www.reddit.com/r/CryptoCurrency/comments/7ra409/your_guide_to_monero_and_why_it_has_great/)_**
+ **_[Introdução técnica ao Monero](https://steemit.com/monero/@sgp/7yjqso-a-monero-introduction-for-beginners)_**
+ **_[Zero to Monero](https://www.getmonero.org/library/Zero-to-Monero-1-0-0.pdf)_**
+ **_[Golpes a serem evitados](https://www.reddit.com/r/Monero/wiki/avoid)_**
+ **_[Monero Perguntas Frequêntes](https://www.getmonero.org/pt-br/get-started/faq/)_**
+ **_[Conecte-se com a Comunidade Monero](https://getmonero.org/community/hangouts/)_**
**_[Mastering Monero](https://masteringmonero.com/)_**